test/controllers/ReviewDetailsControllerSpec.scala

```diff
@@ -324,51 +276,6 @@ class ReviewDetailsControllerSpec extends PlaySpec with OneServerPerSuite with M
             }
           }
 
-          "return agent registration page correctly for Agents" in {
```

Comment:
Are these tests required to be removed or updated with mockNewAgentRegistrationService?